src/modules/home/routes/home.route.tsx

```diff
@@ -1,35 +1,42 @@
-import { AccommodationCard } from '#/modules/home/components/accommodation-card';
 import Box from '@mui/material/Box';
-import Typography from '@mui/material/Typography';
-import CardImage from '#/assets/images/card-temp-image.jpg';
+import { headerHeight } from '#/modules/layout/components/header/header';
+import { useEffect, useState } from 'react';
+import { AccommodationList } from '../components/accommodation-list';
+
+const filterAreaHeight = 60;
+const styles = {
+  filterArea: (showShadow: boolean) => ({
+    minHeight: filterAreaHeight,
+    position: 'fixed',
+    top: `${headerHeight}px`,
+    left: 0,
+    right: 0,
+    zIndex: 10,
+    textAlign: 'center',
+    backgroundColor: '#fff',
+    borderTop: '1px solid #ddd',
+    boxShadow: showShadow ? '0px 4px 8px rgba(0, 0, 0, 0.2)' : 'none',
```

Comment:
Why it is needed?

Reply:
> Why it is needed?

To distinguish the fixed filter area from the accommodations lists when scrolled down